README.md

```diff
@@ -24,14 +31,14 @@ Following VPP metrics are currently supported:
 
 In order to install and run vpptop you need to install following requirements:
  - [Go](https://golang.org/dl/) **1.11**
- - [VPP](https://wiki.fd.io/view/VPP) (**19.04.1** recommended, more info below)
+ - [VPP](https://wiki.fd.io/view/VPP) (**master (19.08)** recommended, more info below)
```

Comment:
This should refer to full version, e.g. `19.08-rc0.574-g063366eb2`. So the users can install the specific version which is certain to be compatible.